packages/graphing/src/__tests__/graph.test.jsx

```diff
@@ -51,6 +51,7 @@ describe('Graph', () => {
     range: { min: 0, max: 1, step: 1 },
     size: { width: 400, height: 400 },
     currentTool,
+    maskId: 'graph-1481',
```

Comment:
If this is random, this will most probably fail. I think you can create a function that generates that id in the file below, and in tests you have to mock it.

Reply:
Hallo Andreea,

Yes, I had this problem and the solution was using props in constructor where we will send the id from snapshot in case of tests.  I was looking on internet too, this is a a general problem for tests when using random id like here.

Reply:
I will merge this as it is now, but as we discussed earlier, let's create a function that we can mock.